tools/replay.cpp

```diff
@@ -553,21 +556,18 @@ class Replay {
         //
         // Need to skip FixedPool for now since I haven't figured out how to
         // dynamically parse/creat the data type parameter
-        //
-        replay_out() << " (ignored) ";
-        return;
-#if 0
         //
         // Replay currently cannot support replaying FixedPool allocations.
         // This is because replay does its work at runtime and the FixedPool
         // is a template where sizes are generated at compile time.
         //
+        replay_out() << " (ignored) ";
+        return;
+      }
+      else if (m_row[3] == "umpire::strategy::MixedPool" ) {
         const std::string& allocName = m_row[6];
-        std::size_t PoolSize = hmm...
-
-        if ( introspection )  m_rm.makeAllocator<umpire::strategy::FixedPool<PoolSize>, true>(name, m_rm.getAllocator(allocName));
-        else                  m_rm.makeAllocator<umpire::strategy::FixedPool<PoolSize>, false>(name, m_rm.getAllocator(allocName));
-#endif
+        if ( introspection ) m_rm.makeAllocator<umpire::strategy::MixedPool, true>(name, m_rm.getAllocator(allocName));
+        else                 m_rm.makeAllocator<umpire::strategy::MixedPool, false>(name, m_rm.getAllocator(allocName));
```

Comment:
Thanks for adding this handler in the replay tool.  Now, we need to please update replay_test to generate one of these pools and then update the output file that the CI uses so that it becomes part of our regular checks.

Reply:
iirc the credit for adding the handler to the replay tool should go to @davidbeckingsale!

Reply:
I'll deal with this after merge.